share/lib/hoc/mknrndll.hoc

```diff
@@ -104,7 +104,7 @@ proc recent_working_dirs() {local i
 
 proc exec() {localobj s1, s2
 	chdir(path)
-    if (sf_.substr(nrnversion(8), "mingw") > 0) {
+    if (sf_.substr(nrnversion(8), "mingw") > 0 || sf_.substr(nrnversion(8), "indows") > 0) {
```

Comment:
`indows` and not `windows`? if so, a comment would be helpful?

Reply:
Just a hack in case sometimes it is windows and sometimes Windows. I am unaware of a case insensitive form of this which I would rather use.